scripts/superdesk-search/directives/ItemList.js

```diff
@@ -1550,6 +1550,13 @@ export function ItemList(
                 },
 
```

Comment:
Hey, I need to use `$broadcast` because I need to affect all scopes of same directive where I will remove selection. `this` will affect only current scope. I will create this `deselectAll` function.